tests/cmd/main_test.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 
 import mock
+from six import StringIO
```

Comment:
Isn't that exactly what `six` does?

I agree with leaning against adding dependencies, but `six` seems to be a pretty standard one, especially for ensuring 2/3 compat